bika/lims/browser/bika_listing.py

```diff
@@ -916,174 +854,159 @@ def folderitem(self, obj, item, index):
         """
         return item
 
-    def folderitems(self, full_objects=False, classic=True):
```

Comment:
I know, wonky again, but please do not miss this `classic` param. Was used to support both lists made up of objects or brains